integration_tests/features/support/steps.js

```diff
@@ -182,6 +182,52 @@ Then('all nodes are at current tip height', {timeout: 1200*1000},async function
     })
 });
 
+Then(/meddling with block template data from node (.*) for wallet (.*) is not allowed/, async function (baseNodeName, walletName) {
+    let baseNodeClient = this.getClient(baseNodeName);
+    let walletClient =this.getWallet(walletName).getClient()
+
+    // No meddling with data
+    // - Current tip
+    let currHeight = await baseNodeClient.getTipHeight();
```

Comment:
nit: Why not use existing steps for lines 191 - 199. It seems to test that mining a block is fine?

Reply:
This is to demonstrate that the new steps introduced are working fine, specifically:
- `baseNodeClient.mineBlockBeforeSubmit`
- `baseNodeClient.submitMinedBlock`